quidel_covidtest/tests/test_run.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 
 from delphi_utils import read_params, add_prefix
-from delphi_quidel_covidtest.constants import GEO_RESOLUTIONS, SENSORS
+from delphi_quidel_covidtest.constants import GEO_RESOLUTIONS, SMOOTHED_POSITIVE
```

Comment:
is there a purpose for this change?

Reply:
Yes. Raw signals are restricted by min_obs. Since the test data is manually generated which is pretty small, it causes output files for raw signals becomes all NAs. So here we only check smoothed signals.

Reply:
ah got it, thanks. is there a way we can still test the raw ones?

Reply:
We can replace the current test data with a larger one which will not make the raw all NAs. Will do.

Reply:
Done.